src/components/DataViewer/DataViewer.tsx

```diff
@@ -103,7 +127,7 @@ const DataViewer: React.FC<DataViewerProps> = (props) => {
                 {props.header}
                 <ClearButton closeModal={closeModal} />
             </Header>
-            <Content>{JSONContent(props.data)}</Content>
+            <Content tabIndex={0}>{JSONContent(props.data)}</Content>
```

Comment:
what's `tabIndex` do?

Reply:
`tabIndex` allows for the modal to be _focusable_ by a user of the website. This is no longer required though as we are changing the modal to use Material UI